src/IRtext.h

```diff
@@ -116,6 +120,7 @@ extern IRTEXT_CONST_PTR(kIndirectStr);
 extern IRTEXT_CONST_PTR(kInsideStr);
 extern IRTEXT_CONST_PTR(kIonStr);
 extern IRTEXT_CONST_PTR(kJkeStr);
+extern IRTEXT_CONST_PTR(kKeyStr);
```

Comment:
This was meant not for a "remote key" or a "command" but rather the setting value (used as `Key:Value` concept).
used as:
```cpp
  case argoIrMessageType_t::CONFIG_PARAM_SET:
    result += addIntToString(_.config.Key, kKeyStr);
    result += addIntToString(_.config.Value, kValueStr);
    break;
```

I can change it to `Setting` to be more descriptive, but this would make it less reusable, I think.
Hence, leaving as-is for now, please advise.

Reply:
Sorry, I missed that. I thought it was used for a button/key-press etc. Ignore my feedback.